src/main/java/mate/academy/springboot/swagger/service/impl/ProductServiceImpl.java

```diff
@@ -0,0 +1,42 @@
+package mate.academy.springboot.swagger.service.impl;
+
+import java.math.BigDecimal;
+import java.util.List;
+import lombok.RequiredArgsConstructor;
+import mate.academy.springboot.swagger.model.Product;
+import mate.academy.springboot.swagger.repository.ProductRepository;
+import mate.academy.springboot.swagger.service.ProductService;
+import org.springframework.data.domain.PageRequest;
+import org.springframework.stereotype.Service;
+
+@Service
+@RequiredArgsConstructor
```

Comment:
You can use AllArgumentConstroctor annotation if you don't have any other initialized fields